app-doc/elisp-manual/Manifest

```diff
@@ -1,14 +1 @@
-DIST elisp-manual-18-1.03.tar.gz 1246385 BLAKE2B af23fd63bf56d5b76785e929aa05e628143d33baf03c30746ee18e8c278113c2874fbe2689c1b361a2d9efd6a92aa6c39a92317f2f36f282f313afa1fc4876bd SHA512 5ba37650482a34918924beafdf214e8beb9adb5e94976020fb8a231d6d5a2dfb4ceb0659172412b193415634a93929462aba4109ebbbef31feca96397a3b2c30
```

Comment:
See other comments re nginx and cleaning up.

Specifically for emacs: these versions correspond to old(er) Emacs slots. It's useful to have an older elisp manual even if it's technically not needed(?) by the Emacs package. So, we shouldn't clean these up, even if the depgraph allows it.